waku/v2/node/waku_node.nim

```diff
@@ -553,6 +548,13 @@ proc mountRelay*(node: WakuNode,
 
   info "relay mounted successfully"
 
+  # TODO: As part of #1545, remove this and update the tests cases
+  node.subscribe(DefaultPubsubTopic)
```

Comment:
This line will be removed as part of a follow-up PR that will resolve #1545

Reply:
Thanks!